src/Authenticator/AuthenticatorCollection.php

```diff
@@ -19,6 +19,9 @@
 use Cake\Core\App;
 use RuntimeException;
 
+/**
+ * @method \Authentication\Authenticator\AuthenticatorInterface|null get(string $name)
```

Comment:
The `|null` doesn't make sense to me, `null` can't have a method `get()`.

Reply:
This comes from the method itself.
As i said: an own wrapper that throws exception would be most clean.